src/Screens/Main/Chat/Title.tsx

```diff
@@ -41,6 +65,22 @@ const Title: React.FC<Props> = ({ style, onPress, name, buddyId }) => {
           style={styles.userIcon}
         />
         <RN.Text style={styles.name}>{name}</RN.Text>
+        {isBanned && (
```

Comment:
This -> _isBanned &&_, is for now used to hide the option to ban the chat? 
Because we dont want to use the ban action yet?

Reply:
Or this was supposed to be only UI pr?

Reply:
This is to not show the ban button when the chat is already banned. Later on, there will be other actions when the chat is banned, i.e. Deleting and unbanning. But for now i decided just to hide the whole dropdown for already banned chats